packages/commerce/src/api/utils/with-operation-callback.ts

```diff
@@ -21,7 +21,7 @@ export const withOperationCallback =
 const parse = ({ name, data }: Operation) => {
   switch (name) {
     case 'getProduct':
-      productSchema.nullable().parse(data.product)
+      productSchema.nullable().parse(data.product ?? null)
```

Comment:
`.nullable()` can be replaced with `.nullish()` so it accepts `null or undefined`, then there is no need for ` ?? null`